code/datums/components/crafting/crafting_lists/misc.dm

```diff
@@ -360,6 +360,15 @@
 	)
 	category = CAT_MISC
 
+/datum/crafting_recipe/steel_sharpener
+	name = "Steel Sharpener"
+	result = /obj/item/sharpener/steel
+	time = 5 SECONDS
+	reqs = list(
+		/obj/item/stack/rods = 2
```

Comment:
This crafting recipe is so cheap its practically pointless to be in the vendor. It needs to be harder, at least needing a plasteel or a diamond.

Reply:
If I make it take a diamond, I will just make it a reskin of the chef stone with the same stats.

Plasteel if you're okay with it being weaker sounds fine

Reply:
I don't mind either